superset/viz.py

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=C,R,W
+# pylint: disable=C,R,W,useless-suppression
```

Comment:
did we give up on this file?

Reply:
@etr2460 no—though it does need a lot of TLC.

I was planning on doing another pass of all files which match `git grep "^pylint: disable=` and remove all top-level—adding them inline where appropriate.